README.md

```diff
@@ -185,6 +185,85 @@ the same directory. This can be helpful if you need to create factories that are
 used for different repos, your factory module is getting too big, or if you have
 different ways of saving the record for different types of factories.
 
+### How to split your factories into different files
+
+We will assume that you use factories only in test environment. If you use them in all environments, just adjust directory structure.
```

Comment:
Done, though the section does not exists yet, suggestions what to put there are welcome.